manifests/charts/istio-control/istio-discovery/templates/validatingwebhookconfiguration.yaml

```diff
@@ -50,36 +50,5 @@ webhooks:
           {{- else }}
           - "{{ .Values.revision }}"
           {{- end }}
-  # Webhook handling default validation
```

Comment:
should we remove the istiod-istio-system one as well now?

Reply:
I still don't think we can because of the resource pruning removing the old, active validator?

Reply:
Aren't we replacing it with a new active validator though? I guess not if we are not the default?

Reply:
I think fixing the old validator is a must-do for 1.11

Reply:
Just tested and it seems like removing the webhook doesn't actually purge it from old installations... should be clear to remove.

Reply:
Should we explicitly purge it when we set the default? That way we don't get a stale webhook, which will either give noisy errors in Istiod at best, or block config at worst?

Reply:
Made it so that (1) validating webhooks get removed on `istioctl uninstall --revision` and (2) we explicitly purge the old one when setting default.